srcpkgs/cawbird/patches/disable-tests.patch

```diff
@@ -0,0 +1,18 @@
+--- tests/meson.build
++++ tests/meson.build
+@@ -1,15 +1,10 @@
+ 
+ tests = [
+   'avatarcache',
+-  'avatardownload',
+   'bundlehistory',
+   'dmmanager',
+-  'filters',
+   'friends',
+-  'inlinemediadownloader',
+-  'texttransform',
+   'tweetmodel',
+-  'tweetparsing',
+   'twitteritem',
+   'usercompletionmodel',
+   'usercounter',
```

Comment:
Can you leave a comment on why these were disabled?

Reply:
I managed to fix the tests